File: game-engine/src/test/java/org/luxons/sevenwonders/game/data/GameDefinitionLoaderTest.java
package org.luxons.sevenwonders.game.data;

import org.junit.Test;

import static org.junit.Assert.assertNotNull;

public class GameDefinitionLoaderTest {

    @Test
    public void successfulLoad() {
        GameDefinitionLoader loader = new GameDefinitionLoader();
        GameDefinition gameDefinition = loader.getGameDefinition();
        assertNotNull(gameDefinition);
    }

}

File: game-engine/src/test/kotlin/org/luxons/sevenwonders/game/data/GameDefinitionLoaderTest.kt
package org.luxons.sevenwonders.game.data

import org.junit.Assert.assertNotNull
import org.junit.Test

class GameDefinitionLoaderTest {

    @Test
    fun successfulLoad() {
        val gameDefinition = GameDefinitionLoader.gameDefinition
        assertNotNull(gameDefinition)
    }
}
